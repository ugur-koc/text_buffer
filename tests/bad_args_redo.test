redo bad_arg
exit