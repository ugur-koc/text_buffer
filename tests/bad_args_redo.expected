Available Operations: insert append eraseAt erase replace load save help exit 
Text Buffer: 
Arguments to redo are ignored!

Nothing to redo

Text Buffer: 
